### DISCUSSION TITLE
[Feature] Add more options to control Partner Sharing features in user settings + expose external library content in other views

### DISCUSSION THREAD

**Discussion by dgtal1** [immich-app/immich#9075] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The current design of external library is a bit awkward. Admin defines external libraries, and who to share them with. This is fine. But then users don't see everything what admin can see. They can only see the external library contents in the Photos view / timeline, but no faces, no places.

I propose to enhance the control over the External Library content seen by users and extend it. I'm talking about this:
![image](https://github.com/immich-app/immich/assets/27864579/28a09d53-59e3-40d0-9e5c-9f536a3ec38b)

Additional controls proposed:
* Places in the Explore view (including showing these photos on the map)
* People view (including People shown in the Explore view)

I propose these to be `on` by default

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5089
